src/Tools/Test Framework/Test Runner/src/DataDrivenTest/DataInputs/TestInputsManagement.Codeunit.al

```diff
@@ -153,15 +160,24 @@ codeunit 130458 "Test Inputs Management"
         ParseDataInputs(DataInputText, TestInputGroup);
     end;
 
-    local procedure CreateTestInputGroup(var TestInputGroup: Record "Test Input Group"; FileName: Text)
+    local procedure CreateTestInputGroup(var TestInputGroup: Record "Test Input Group"; FileName: Text; ImportedByAppId: Guid)
+    var
+        FileNameLengthErr: Label 'File name length exceeds 100 characters';
+        EmptyGuid: Guid;
     begin
-#pragma warning disable AA0139
-        TestInputGroup.Code := FileName;
+        if StrLen(FileName) > 100 then
```

Comment:
No hardcoding, it is hardcoded in 3 places. File name should follow the OS limitation for the file name.

Reply:
Changed the way we evaluate TestInputGroup.Code from Filename.

Reply:
Removed the checks and new field.